src/mirage_protocols.mli

```diff
@@ -236,9 +236,9 @@ module type ICMP = sig
   (** [input t src dst buffer] reacts to the ICMP message in
       [buffer]. *)
 
-  val write : t -> dst:ipaddr -> buffer -> (unit, error) result io
-  (** [write t dst buffer] sends the ICMP message in [buffer] to [dst]
-      over IP. *)
+  val write : t -> dst:ipaddr -> ?ttl:int -> buffer -> (unit, error) result io
+  (** [write t dst ?ttl buffer] sends the ICMP message in [buffer] to [dst]
```

Comment:
```suggestion
  (** [write t dst ~ttl buffer] sends the ICMP message in [buffer] to [dst]
```
the `[ ... ]` is a call-site of a function, where usually `~` is used for labeled arguments.